src/components/articles/articleCard.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+const ArticleCard = ({ article, onClick }) => {
+  return (
+    <div id={article.slug}>
+      <img
+        src='../../assets/images/pexels-photo-46710.jpeg'
```

Comment:
Can `import Pexels from '../../assets/images/pexels-photo-46710.jpeg'` then use `src={Pexels}` here. it will help with issue of images that are not being displayed on heroku deployment